spec/features/project/show_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "Project Show", type: :feature do
+  describe "As a visitor" do
+    scenario "when I visit the show page I should see" do
```

Comment:
Nice job with this test organization. I like the "As a visitor" part and then the scenario. Great!